data_pipeline/gmail_data/get_gmail_data.py

```diff
@@ -103,4 +104,9 @@ def get_distinct_values_from_bq(
     query_job = client.query(sql)  # API request
     results = query_job.result()  # Waits for query to finish
 
-    return pd.concat([pd.DataFrame([row.threadId]) for row in results])
+    return pd.concat([pd.DataFrame([row.threadId]) for row in results], ignore_index=True)
+
+
+def dataframe_chunk(seq, size):
```

Comment:
this would be a lovely function to unit test :-)

Reply:
:)